fe/fe-core/src/test/resources/sql/subquery/in-subquery.sql

```diff
@@ -886,3 +886,101 @@ NULL AWARE LEFT ANTI JOIN (join-predicate [19: cast = 20: abs AND add(cast(9: t1
         SCAN (columns[7: t1a, 9: t1c, 10: t1d] predicate[null])
 [end]
 
+[sql]
```

Comment:
add test to cover (int, String) in (string, double)....more implict cast case

Reply:
done